### DISCUSSION TITLE
First time user doubts

### DISCUSSION THREAD

**Discussion by Bhaalspawn** [mpvnet-player/mpv.net#441] [Q&A]
Hello i'm new to mpv.net
I'm trying this player and i see there are few things i need...setting are a bit confusing so i don't know there is already what i need or not.
Not big features but just someuseful things

1) on top ONLY WHILE PLAYING
2) show OSC only when mouse cursor reach the bottom of the player
3) Hide OSD (i don't need those bars when i change volume or seek in the file)

**Comment by stax76** [comment]
You can perform a text search in the command palette (F1) and also perform a text search in the manual.

Learning mpv takes time, you can ask questions here:

https://www.reddit.com/r/mpv/

**Reply by Bhaalspawn** [reply]
ok but i was looking for a gui with easy settings.here it seems to have to do everything manually

for point 1 i think i found this
https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/ontop-playback.lua

but i don't know what to do with it.

i didn't even have to install mpv.net so i don't have any folder

**Comment by hooke007** [comment]
1.
- If using [ontop-playback.lua](https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/ontop-playback.lua)
To create the folder by your self https://mpv.io/manual/master/#script-location
https://mpv.io/manual/master/#files-on-windows
For mpvnet, the dir is `C:\users\USERNAME\AppData\Roaming\mpv.net\scripts\ontop-playback.lua`
- Or just try [profiles](https://mpv.io/manual/master/#profiles)
```
[ontop_playback]
 profile-cond = pause
 ontop        = no

[ontop_playback-alt]
 profile-cond = not pause
 ontop        = yes

```

2.
https://mpv.io/manual/master/#on-screen-controller-deadzonesize
To set `deadzonesize=1` . Note it is a script-opt (https://mpv.io/manual/master/#configuration).

3.
To set `osd-bar=no` in mpv.conf
https://mpv.io/manual/master/#options-no-osd-bar

P.S.
> a gui with easy settings.

So far I haven't see any frontend could reach the goal.

**Reply by Bhaalspawn** [reply]
you have to understand this kind of help is useless for me. please explain step by step..i understood what you wrote but for someone like me this is incomplete.

2 Ok i understood what deadzonesize is but i don't see any setting about it in config editor...then you wrote "script-opt"...i have this folder in appdata/etc etc and inside i have osc.conf and console.conf (i can't open those files)

3 Set where????? again there is no such setting in config editor. in "screen" i see osd-playing-msg, osd-font-size osd-duration and osd-scale-by-window.......besides you wrote "osd-bar=no" but in the link they say "--no-osd-bar". this is a minor problem because is could try both lines but it makes things more cunfusing

**Reply by hooke007** [reply]
All the things is clear in the official manual, but if you don’t have time to read full of the contents —
Quickly review this link https://github.com/Argon-/mpv-config to assist you in understanding the structure of mpv-conf folder.

> 2. …(i can't open those files)

open them via notepad.

> 3. …. Set where

I have said “mpv.conf”, a file should be located in `C:\users\USERNAME\AppData\Roaming\mpv.net\mpv.conf`. To add the contents I mentioned maually.

> this is a minor problem because is could try both lines but it makes things more cunfusing

ref https://mpv.io/manual/master/#usage
In mpv.conf, `no-osd-bar` is equivalent to `osd-bar=no`

> no such setting in config editor

For there is a performance limitation in editor, currently it is impossible to include all the options.

**Reply by Bhaalspawn** [reply]
ok thank i did 2 and 3

the first was more difficult to understand..

"Scripts can be passed to the --script option" i don't know what it means "and are automatically loaded from the scripts subdirectory of the mpv configuration directory (usually ~/.config/mpv/scripts/)." so it's mpv not mpv.net?? it's confusing if i have to check 2 different folder... 
So i went to appdata/roaming/mpv and i see it's empty
I created "script" folder there and downloaded the script but it's not working.

i think it's because this folder is empty and i'm using a portable version or mpv because i didn't find any installer.
i tried to download it from microsoft store but that version not working..i couldn't even open settings or anyuthing else.

**Reply by hooke007** [reply]
I have said
> 1. ...For mpvnet, the dir is `C:\users\USERNAME\AppData\Roaming\mpv.net\scripts\ontop-playback.lua`

You are using mpvnet so the conf dir is `\mpv.net\` instead of `\mpv\`

BTW, you also need to set `ontop=yes` in mpv.conf. (If you decide to use the ontop-playback.lua)
